index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8" />
+    <title>Hello React Typescript!</title>
+</head>
+
+<body>
+    <div id="demo-app"></div>
+
+    <!-- Dependencies -->
```

Comment:
Is this really needed? I don't think this is needed because without this it should work.

Reply:
Dependencies??

Reply:
Yes. both script links.

Reply:
Ok! Let me check !!